src/wallet/rpcwallet.cpp

```diff
@@ -3736,32 +3736,56 @@ UniValue getaddressinfo(const JSONRPCRequest& request)
     UniValue ret(UniValue::VOBJ);
     CTxDestination dest = DecodeDestination(request.params[0].get_str());
 
-    // Make sure the destination is valid
+    // Ensure the destination is valid.
     if (!IsValidDestination(dest)) {
         throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, "Invalid address");
     }
 
+    // Return address field.
```

Comment:
1d6884d4d4b541d0c04ef10da208ef657122aa1f

IMO these "return foo field" comments are useless.

Reply:
Thank you for reviewing, @promag. Some of the comments add info that isn't obvious and the others are for consistency and separation. I propose to keep the 3 tested ACKs here and remove too-obvious comments in the follow-up (to add multiple labels / deprecate the label field).

Reply:
Had to rebase, so removed the obvious comments.